ansible/roles/debops.tinc/templates/etc/tinc/network/tinc-up.j2

```diff
@@ -115,7 +117,7 @@ else
     fi
 
     if [ -n "${accept_ra}" ]; then
-        sysctl net.ipv6.conf.${INTERFACE}.accept_ra=${accept_ra}
+        echo -n "${accept_ra}" > "/proc/sys/net/ipv6/conf/${INTERFACE}/accept_ra"
```

Comment:
Why this change? I think that using `sysctl` command for this instead of `echo`ing the value directly is better by using the "official" API. Just quote the `net.ipv6...$${accept_ra}` and you should be good.

I assume that the value is removed automatically by the kernel when the interface goes out?